src/ResourceManager/PowerBIDedicated/Commands.PowerBIDedicated/Commands/UpdateAzureRmPowerBIEmbeddedCapacity.cs

```diff
@@ -0,0 +1,125 @@
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+using System;
+using System.Collections;
+using System.Linq;
+using System.Management.Automation;
+using Microsoft.Azure.Commands.PowerBIDedicated.Models;
+using Microsoft.Azure.Commands.PowerBIDedicated.Properties;
+using Microsoft.Azure.Commands.PowerBIDedicated.Utilities;
+using Microsoft.Azure.Commands.ResourceManager.Common.Tags;
+using Microsoft.Azure.Management.PowerBIDedicated.Models;
+
+namespace Microsoft.Azure.Commands.PowerBIDedicated
+{
+    [Cmdlet(VerbsData.Update, "AzureRmPowerBIEmbeddedCapacity", SupportsShouldProcess = true, DefaultParameterSetName = ParamSetDefault), OutputType(typeof(AzurePowerBIDedicated))]
+    public class UpdateAzurePowerBIEmbeddedCapacity : PowerBIDedicatedCmdletBase
+    {
+        private const string ParamSetDefault = "Default";
+
+        [Parameter(ValueFromPipelineByPropertyName = true, Position = 0, Mandatory = true,
+            HelpMessage = "Name of the capacity.")]
+        [ValidateNotNullOrEmpty]
+        public string Name { get; set; }
+
+        [Parameter(ValueFromPipelineByPropertyName = true, Position = 1, Mandatory = false,
+            HelpMessage = "Name of resource group under which you want to update the capacity.")]
+        [ValidateNotNullOrEmpty]
+        public string ResourceGroupName { get; set; }
+
+        [Parameter(ValueFromPipelineByPropertyName = true, Position = 2, Mandatory = false,
+            HelpMessage = "Name of the Sku used to create the capacity")]
+        [ValidateNotNullOrEmpty]
+        [ValidateSet("A1", "A2", "A3", "A4", "A5", "A6", IgnoreCase = true)]
+        public string Sku { get; set; }
+
+        [Parameter(ValueFromPipelineByPropertyName = true, Position = 3, Mandatory = false,
+            HelpMessage = "A string,string dictionary of tags associated with this capacity")]
+        [ValidateNotNull]
+        public Hashtable Tag { get; set; }
+
+        [Parameter(ValueFromPipelineByPropertyName = true, Position = 4, Mandatory = false,
+            HelpMessage = "A comma separated capacity names to set as administrators on the capacity")]
+        [ValidateNotNull]
+        public string[] Administrator { get; set; }
+
+        [Parameter(Mandatory = false, ValueFromPipelineByPropertyName = true,
```

Comment:
@selasagi this should be the only parameter that has `ValueFromPipelineByPropertyName = true`

Reply:
Done